Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/FarePolicy.hs

```diff
@@ -122,8 +123,9 @@ updateFarePolicy admin fpId req = do
           } ::
           DFarePolicy.FarePolicy
   coordinators <- QP.findAdminsByMerchantId admin.merchantId
+  let deletedBy = EsqDE.DeletedByPerson $ cast @SP.Person @EsqDE.Person admin.id
   Esq.runTransaction $
```

Comment:
You can just pass `deletedBy` (or rather `updateRequestor` i guess) as part of `runTransaction`, then you won't need to pass it to every single delete. Can you do that?